streaming/redpanda/addons.tf

```diff
@@ -0,0 +1,191 @@
+    }
+    kube-proxy = {
+      most_recent = true
+    }
+  }
+  enable_aws_load_balancer_controller    = true
+  enable_cluster_autoscaler              = true
+  enable_metrics_server                  = true
+  enable_aws_cloudwatch_metrics          = true
+
+ 
+
+  #---------------------------------------
+  # FluentBit Config for EKS Cluster
+  #---------------------------------------
+ enable_aws_for_fluentbit = true
+  aws_for_fluentbit = {
+    enable_containerinsights = true
+    kubelet_monitoring       = true
+    set = [{
+        name  = "cloudWatchLogs.autoCreateGroup"
+        value = true
+      },
+      {
+        name  = "hostNetwork"
+        value = true
+      },
+      {
+        name  = "dnsPolicy"
+        value = "ClusterFirstWithHostNet"
+      }
+    ]
+  }
+
+
+  #---------------------------------------
+  # Prommetheus and Grafana stack
+  #---------------------------------------
+  #---------------------------------------------------------------
+  # Install Kafka Montoring Stack with Prometheus and Grafana
+  # 1- Grafana port-forward `kubectl port-forward svc/kube-prometheus-stack-grafana 8080:80 -n kube-prometheus-stack`
+  # 2- Grafana Admin user: admin
+  # 3- Get admin user password: `aws secretsmanager get-secret-value --secret-id <output.grafana_secret_name> --region $AWS_REGION --query "SecretString" --output text`
+  #---------------------------------------------------------------
+  enable_kube_prometheus_stack = true
+  kube_prometheus_stack = {
+    values = [
+      var.enable_amazon_prometheus ? templatefile("${path.module}/templates/kube-prometheus-amp-enable.yaml", {
+        region              = local.region
+        amp_sa              = local.amp_ingest_service_account
+        amp_irsa            = module.amp_ingest_irsa[0].iam_role_arn
+        amp_remotewrite_url = "https://aps-workspaces.${local.region}.amazonaws.com/workspaces/${aws_prometheus_workspace.amp[0].id}/api/v1/remote_write"
+        amp_url             = "https://aps-workspaces.${local.region}.amazonaws.com/workspaces/${aws_prometheus_workspace.amp[0].id}"
+        storage_class_type  = kubernetes_storage_class.ebs_csi_encrypted_gp3_storage_class.id
+      }) : templatefile("${path.module}/templates/kube-prometheus.yaml", {})
+    ]
+    chart_version = "48.1.1"
+    set_sensitive = [
+      {
+        name  = "grafana.adminPassword"
+        value = data.aws_secretsmanager_secret_version.grafana_password_version.secret_string
+      }
+    ],
+  }
+
+  tags = local.tags
+}
+
+ ## Cert-Manager Config
+  resource "helm_release" "cert-manager" {
+    name = "cert-manager"
+    repository = "https://charts.jetstack.io"
+    chart = "cert-manager"
+    version = "1.14.2"
+    namespace = "cert-manager"
+    create_namespace = true
+    set {
+     name = "installCRDs"
+     value = true 
+    }
+    depends_on = [ module.eks_blueprints_addons ]
+  }
+  
+  ## Redpanda Helm Config
+resource "helm_release" "redpanda" {
```

Comment:
Redpanda needs to be added in eks_data_addons, https://github.com/aws-ia/terraform-aws-eks-data-addons